demo-extended/src/main/java/com/novoda/downloadmanager/demo/extended/DemoApplication.java

```diff
@@ -28,10 +32,30 @@ public boolean isAllowedToDownload(Download download) {
     }
 
     @Override
-    public NotificationCustomiser getNotificationCustomiser() {
+    public QueuedNotificationCustomiser getQueuedNotificationCustomiser() {
         return notificationCustomiser;
     }
 
+    @Override
+    public DownloadingNotificationCustomiser getDownloadingNotificationCustomiser() {
+        return notificationCustomiser;
+    }
+
+    @Override
+    public CompleteNotificationCustomiser getCompleteNotificationCustomiser() {
+        return null;
```

Comment:
we are now going to change the public interface so that the user doesn't have to implement these methods if they don't want to i.e. no returning null yayyay